dir.props

```diff
@@ -81,7 +81,8 @@
   <PropertyGroup>
      <ArmEnvironmentVariables Condition="'$(ArmEnvironmentVariables)' == ''">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/arm</ArmEnvironmentVariables>
      <ArmEnvironmentVariables Condition="'$(Platform)' == 'armel'">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/armel</ArmEnvironmentVariables>
-   </PropertyGroup>
+     <CrossRootFsEnvironmentVariables Condition="'$(Platform)' == 'x86'">ROOTFS_DIR=$(BaseIntermediatePath)crossrootfs/x86</CrossRootFsEnvironmentVariables> 
```

Comment:
It seems like it would be cleaner to rename all three of these to `RootFsEnvironmentVariables` so `ROOTFS_DIR` can be set more uniformly in all the build commands.

Reply:
Yep, I have the same idea but didn't want to rename things. 
So set everything as `<RootFsEnvironmentVariables>` ?

Reply:
Doing a search, I don't see any places a rename would affect other than the parts you're already making changes near, so I'd say it'd be good to make this refactor now. Without CI testing x86 or ARM it's hard to know for sure how fragile this is, though....

I'm ok leaving it like you have it now. The other comments (`BuildArguments` duplication) seem to have a bigger impact on maintainability, anyway.

Reply:
I have renamed all cross related EnvironmentVariables to RootFsEnvironmentVariables
commit: #9f1a856